htdocs/expensereport/card.php

```diff
@@ -2559,7 +2559,7 @@
 
 				// Quantity
 				print '<td class="right inputqty">';
-				print '<input type="text" min="0" class="right maxwidth50" name="qty" value="'.dol_escape_htmltag($qty ? $qty : 1).'">'; // We must be able to enter decimal qty
+				print '<input type="text" min="0" class="right maxwidth50" id="input_qty" name="qty" value="'.dol_escape_htmltag($qty ? $qty : 1).'">'; // We must be able to enter decimal qty
```

Comment:
A field with id="input_qty" has already be added line 2300.
an html id must be used only once per page.

Reply:
maybe i'm wrong but this two ids declaration never appear at the same moment.
$action != 'editline'  in one case and $action == 'editline' for the other

Reply:
i made the changes on my last commit.